armi/reactor/parameters/parameterDefinitions.py

```diff
@@ -575,6 +575,10 @@ def toWriteToDB(self, assignedMask: Optional[int] = None):
         """
         Get a list of acceptable parameters to store to the database for a level of the data model.
 
+        .. impl:: Implementation for restricting some parameters from being written to the database.
+            :id: I_ARMI_RESTRICT_DB_WRITE
+            :links: R_ARMI_RESTRICT_DB_WRITE
```

Comment:
I think we decided we're not doing `..impl::` tags after all.

Reply:
@john-science as I understand it and last I heard, they were optional. Has this changed recently?

Reply:
@albeanth If the team isn't doing these at large, I don't want to be the one team comiting to the extra week(s) worth of work to do the implementation crumbs.